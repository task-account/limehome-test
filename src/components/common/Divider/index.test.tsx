import renderer from "react-test-renderer";
import { Divider } from ".";

describe(`Divider`, () => {
  it("render", () => {
    const tree = renderer.create(<Divider vertical={"l"} />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("vertical", () => {
    const tree = renderer.create(<Divider vertical={"l"} />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("horizontal", () => {
    const tree = renderer.create(<Divider horizontal={"l"} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
