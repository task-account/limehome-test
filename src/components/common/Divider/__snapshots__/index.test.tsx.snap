// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider horizontal 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#4b4b4d",
      "height": 1,
      "marginBottom": undefined,
      "marginHorizontal": 24,
      "marginLeft": undefined,
      "marginRight": undefined,
      "marginTop": undefined,
      "marginVertical": undefined,
    }
  }
/>
`;

exports[`Divider render 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#4b4b4d",
      "height": 1,
      "marginBottom": undefined,
      "marginHorizontal": undefined,
      "marginLeft": undefined,
      "marginRight": undefined,
      "marginTop": undefined,
      "marginVertical": 24,
    }
  }
/>
`;

exports[`Divider vertical 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#4b4b4d",
      "height": 1,
      "marginBottom": undefined,
      "marginHorizontal": undefined,
      "marginLeft": undefined,
      "marginRight": undefined,
      "marginTop": undefined,
      "marginVertical": 24,
    }
  }
/>
`;
